packages/path-utils/src/index.ts

```diff
@@ -3,13 +3,20 @@ import path from 'path';
 /**
  * Returns true if the parent path contains the child path. Used to allow code to make checks that
  * prevent directory traversal attacks.
- * @param parentPath The path of the parent directory. Assumed to be absolute.
- * @param childPath The path of the child directory. Usually absolute, but if relative, resolved in relation to the parent directory.
+ * @param parentPath The path of the parent directory. If provided, assumed to be absolute. If set to null, the child path is normalized and checked against an arbitrary parent directory.
```

Comment:
I don't actually know what it means for a path to be "checked against an arbitrary parent directory". It feels like we're trying to make `contains` do something it's not meant to do; I also worry that allowing null values here makes it easier for someone to use this function incorrectly.

Do we need another function that performs a different kind of check, ideally one that only takes a single `string` argument instead of two?

Reply:
Created a separate `isContainedRelativePath` function in f39a4407381ce3f6eccdb2f31a9639e9655fefeb.